static/config/test-data.json

```diff
@@ -11,6 +11,7 @@
     "Address": "0xA0a19221268d939c3a972bf5461dC10f7980E814"
   },
   "Account3": {
-    "Address": "0xa7e0214f35481b644e139d689db54b3e625f1c88"
+    "Address": "0x3ad34e698d4806afd08b359b920f5c6b62b68ee4",
+    "Password": "password"
```

Comment:
I suppose password shouldn't be here?

Reply:
I need to store the password somewhere so `geth/account/accounts_test.go` `TestVerifyAccountPasswordWithAccountBeforeEIP55` can run w/o access to the `ACCOUNT_PASSWORD` env variable. It seemed like the json file was a better place to store the password then hardcoding it in a function.

Reply:
Yeah but what I mean is that it had better be in `status-chain-accounts.json` right? By following the logic that accounts in that file are supposed to be run with a hardcoded password while these should run with a password from an env variable.

Reply:
ah, I see. If I do that, then `TestConfig.Account3` will be nil when running tests on the public chain. I don't think that'll break anything, as only the unit tests use `Account3`.

If that's okay, then maybe I should pull the accounts out of that file entirely. Place `Account1` & `Account2` in a `public-chain-accounts.json` & `Account3` in `status-chain-accounts.json`. That feels like a cleaner solution then having some accounts in `test-data.json` and then overriding those accounts when on `StatusChain`.

Reply:
Sounds good. Please, do.